apps/server/src/controllers/paymentsController/fundGroupSavingsWallet.ts

```diff
@@ -0,0 +1,122 @@
+import { Response } from 'express'
+import { Op } from 'sequelize'
+import { HTTP_STATUS_CODE } from '../../constants'
+import { RequestExt } from '../../middleware/authorization/authentication'
+import Wallets, { WalletType, OwnerType } from '../../models/wallets'
+import { fundGroupWalletSchema } from '../../utils/validators'
+import {
+  transactionStatus,
+  transactionType,
+  action,
+} from '../../models/transactions'
+import { createTransaction } from '../../utils/helpers'
+
+export const fundPersonalGroupWallet = async (
+  req: RequestExt,
+  res: Response
+) => {
+  try {
+    const { _userId: userId, ...rest } = req.body
+    const verifyAmount = { amount: rest.amount, email: rest.email }
```

Comment:
Why do you need an email to fund the group wallet? I can't see where the email is required in the Figma.